Tools/parametric_model/configs/fixedwing_singularityfree_model.yaml

```diff
@@ -0,0 +1,147 @@
+# general information
+model_name: "Gazebo Standard Plane"
+model_type: "Standard Plane Longitudinal Model"
+model_class: "GlobalSingularityFreeModel"
+
+extractor_class: "SingularityFreeExtractorModel"
+extractor_config:
+  vmin: 8.0
+  vmax: 20.0
+
+  # all vectors in FRD body frame if not specified otherwise
+model_config:
+  mass: 1.45
+  moment_of_inertia:
+    Ixx: 0.107
+    Iyy: 0.256
+    Izz: 0.356
+  actuators:
+    rotors:
+      # All rotors in the same group will share the coefficients
+      puller_:
+        - rotor:
+          description: "puller rotor"
+          rotor_type: "LinearRotorModel"
+          dataframe_name: "throttle"
+
+  aerodynamics:
+    area: 0.41
+    chord: 0.19
+    stall_angle_deg: 20
+    sig_scale_factor: 30
```

Comment:
This doesn't necessarily configure which aerodynamics model is being used in the model, right?

Reply:
No it shouldn't. The model is defined through the model_class prop at the top of the file, if I implemented it correctly? And the extractor_class configures the corresponding extractor

Reply:
Right, I think lets merge it as is and fix things later :)